jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/basic/BasicFiltersView.java

```diff
@@ -61,5 +61,12 @@ void addDataSetSelectFilter(String label,
                                 String valueColumnId,
                                 Consumer<ActiveFilterItem<String>> callback);
 
+    void addDataSetSelectFilter(String label,
```

Comment:
why not use _addSelectFilter_? Seems more appropriate for a static list of values like this. DataSetSelectFilter is based on dynamic data from db.

Reply:
updated, Please review again.